chat.html

```diff
@@ -134,10 +149,23 @@
       const defaultLanguage = 'en-US';
       const initialLanguage = defaultLanguage;
 
+      //Maybe put helpline strings in a file of constants to avoid misspellings
```

Comment:
Good idea.  Probably unnecessary when we just have this single file, but worth doing when we build more.